github/resource_github_organization_webhook.go

```diff
@@ -23,12 +22,6 @@ func resourceGithubOrganizationWebhook() *schema.Resource {
 		MigrateState:  resourceGithubWebhookMigrateState,
 
 		Schema: map[string]*schema.Schema{
-			"name": {
```

Comment:
Is this worth deprecating with a note saying it's been removed from the api?

Reply:
You'll want to remove this attribute from the docs as well

Reply:
It was already not really passed with only a single valid value, so figured it can just be removed now. I agree ideally it should have been deprecated, but at this point I think a major version bump is fine.